content-based-message-routing/main.bal

```diff
@@ -0,0 +1,92 @@
+import ballerina/http;
+import ballerina/log;
+
+configurable int port = 8290;
+
+final http:Client grandOakEP = check initializeHttpClient("http://localhost:9090/grandoaks/categories");
+final http:Client clemencyEP = check initializeHttpClient("http://localhost:9090/clemency/categories");
+final http:Client pineValleyEP = check initializeHttpClient("http://localhost:9090/pinevalley/categories");
+
+function initializeHttpClient(string url) returns http:Client|error => new (url);
+
+type Patient record {|
+    string name;
+    string dob;
+    string ssn;
+    string address;
+    string phone;
+    string email;
+|};
+
+type ReservationRequest record {|
+    Patient patient;
+    string doctor;
+    string hospital_id;
+    string hospital;
+    string appointment_date;
+|};
+
+type Doctor record {|
+    string name;
+    string hospital;
+    string category;
+    string availability;
+    float fee;
+|};
+
+type ReservationResponse record {|
+    int appointmentNumber;
+    Doctor doctor;
+    Patient patient;
+    float fee;
+    string hospital;
+    boolean confirmed;
+    string appointmentDate;
+|};
+
+enum HospitalIds {
+    GRANDOAKS = "grandoaks",
```

Comment:
Shouldn't this be Grand Oak?

Anyway

```suggestion
    GRAND_OAKS = "grandoaks",
```

Reply:
Kept the same hospital ID we had in the APIm tutorial. However, I'll change this in all the PRs since we don't have any information on hospital id